index.html

```diff
@@ -30,6 +30,70 @@ <h1 class="first-screen__title">Мы создаем легкие решения
                     </div>                
                 </article>
         </section>
+        <section class="services">
+            <div class="services__container">
+                <header class="services__header">
+                    <h2 class="title services__title ">
```

Comment:
```suggestion
                    <h2 class="services__title title">
```
краще клас елементу писати першим, після клас іншого блоку